### PR TITLE
REFACTOR layout and REMOVE usplit component

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: the way the split drag handles worked was quite flacky

## What I did
I removed the additional component for the handle, and made them part of the react-split Resizer component css class.

slight regression in theming (handles are no longer themable) might find a fix soon.

### CODE REVIEW

**Comment by ndelangen** [comment]
Please let me know if you think the non-theme-ability of the 2 grey bars in the handle is an breaking-issue for you.

Personally I think I can fix it a bit later in another PR.

File: lib/components/src/layout/desktop.js [hunk 1]
```diff
@@ -42,21 +91,18 @@ const AddonPanelWrapper = styled('div')(
   })
 );
 
-const resizerCursor = isVert => (isVert ? 'col-resize' : 'row-resize');
+// const resizerStyles = (isShown, isVert) => ({
```
Comment (igor-dv) [inline lib/components/src/layout/desktop.js]: Do you need all these comments later?

File: lib/components/src/layout/desktop.js [hunk 2]
```diff
@@ -2,15 +2,64 @@ import React, { Component } from 'react';
...
+import { injectGlobal } from 'emotion';
 import styled from 'react-emotion';
 import throttle from 'lodash.throttle';
 
 import SplitPane from 'react-split-pane';
 
-import USplit from './usplit';
 import Dimensions from './dimensions';
 
+// eslint-disable-next-line no-unused-expressions
```
Comment (igor-dv) [inline lib/components/src/layout/desktop.js]: 🤦‍♂️

**Review by igor-dv — COMMENTED** [review]
IMO, the theming could be done in a separate PR indeed

**Review by danielduan — APPROVED** [review]

File: lib/components/src/layout/desktop.js (hunk 2 above)
Comment (ndelangen) [reply lib/components/src/layout/desktop.js]: Well it makes sense, it's a deliberate side-effect..

File: lib/components/src/layout/desktop.js (hunk 1 above)
Comment (ndelangen) [reply lib/components/src/layout/desktop.js]: No I will remove them

**Comment by ndelangen** [comment]
Would love your approval @igor-dv 🙇

**Review by igor-dv — APPROVED** [review]
✌

**Pull request merged**